johnsonandjohnson/Bodiless-JS
# feat: Vital Content Listing

## Changes
- Create vital content listing package and content listing variation of generic template
- Small changes to bodielss filtering: Query parameters do not override default filters
- 
## Test Instructions
There are 3 stleguide pages
- content-listing - uses site-wide data, single allowed filter, no default filters
- content-listing-2 - use site-wide data, single alllowed filter, default filter set
- contont-listing-multiple - uses page-local data, multiple allowed filters, no default filter selector
Also - the styleguide page is replaced with a content listing page using site-wide data.

To verify:
- reset button is removed and fitlers are disbled when a default filter is specified
- query parameters are  overridden when a default filter is specified (they should be set to what is defined by the default filter)
-
